src/components/dar_collection_table/DarCollectionTable.js

```diff
@@ -68,61 +68,113 @@ export const styles = {
   },
 };
 
-const columnHeaderFormat = {
-  darCode: {label: 'DAR Code', cellStyle: { width: styles.cellWidth.darCode }},
-  name: {label: 'Project Title', cellStyle: { width: styles.cellWidth.projectTitle }},
-  submissionDate: {label: 'Submission Date', cellStyle: {width: styles.cellWidth.submissionDate }},
-  pi: {label: 'PI', cellStyle: {width: styles.cellWidth.pi }},
-  institution: {label: 'Institution', cellStyle: { width: styles.cellWidth.institution }},
-  datasetCount: {label: 'Datasets', cellStyle: { width: styles.cellWidth.datasetCount }},
-  status: {label: 'Status', cellStyle: {width: styles.cellWidth.status }},
-  actions: {label: 'Action', cellStyle: { width: styles.cellWidth.actions }}
+export const DarCollectionTableColumnOptions = {
+  DAR_CODE: 'darCode',
+  NAME: 'name',
+  SUBMISSION_DATE: 'submissionDate',
+  PI: 'pi',
+  INSTITUTION: 'institution',
+  DATASET_COUNT: 'datasetCount',
+  STATUS: 'status',
+  ACTIONS: 'actions'
 };
 
-const columnHeaderData = () => {
-  //TODO: signing official console should not have an actions column
-  const {darCode, name, submissionDate, pi, institution, datasetCount, status, actions} = columnHeaderFormat;
-  return [darCode, name, submissionDate, pi, institution, datasetCount, status, actions];
+const columnHeaderConfig = {
+  darCode: {
+    label: 'DAR Code',
+    cellStyle: { width: styles.cellWidth.darCode },
+    cellDataFn: cellData.darCodeCellData,
+    sortable: true
+  },
+  name: {
+    label: 'Project Title',
+    cellStyle: { width: styles.cellWidth.projectTitle },
+    cellDataFn: (props) => {
+      props.projectTitle = getProjectTitle(props.collection);
+      return cellData.projectTitleCellData(props);
+    },
+    sortable: true
+  },
+  submissionDate: {
+    label: 'Submission Date',
+    cellStyle: { width: styles.cellWidth.submissionDate },
+    cellDataFn: cellData.submissionDateCellData,
+    sortable: true
+  },
+  pi: {
+    label: 'PI',
+    cellStyle: { width: styles.cellWidth.pi },
+    cellDataFn: cellData.datasetCountCellData,
```

Comment:
I'm guessing this is a copy-paste holdover? It should be the PI data

Reply:
Oh goodness, thank you, this is a great catch! XD Yes, there is already a piCellData method, I'll update it to use that 👍